di-ipv-ticf-stub/lambdas/package.json

```diff
@@ -11,6 +11,7 @@
     "@aws-sdk/lib-dynamodb": "^3.460.0",
     "esbuild": "0.19.8",
     "jose": "^5.6.3",
+    "prettier": "^3.3.3",
```

Comment:
Do we need prettier to be a full dependency? I'd expect it to be a dev dependency

Reply:
Ah yep, good shout